templates/cookie-notice.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Netzstrategen\CoreStandards;
+
+?>
+
+<style>
+  #cookie-notice {
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    z-index: 100000;
+    padding: 0.5em;
+    text-align: center;
+    background-color: #000;
+    color: #fff;
+  }
+
+  <?= do_action('core_standards/cookie_notice/css') ?>
```

Comment:
The hook name `css` is the only sensible name for a filter (in case we add a filter in the future), so I would use a more action-y name like `after_css` or `css_extend`:
```suggestion
  <?= do_action('core_standards/cookie_notice/css_extend') ?>
```